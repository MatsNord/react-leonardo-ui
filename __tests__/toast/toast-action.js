import React from 'react';
import renderer from 'react-test-renderer';

import Toast from '../../src/toast';

test('Toast action renders correctly', () => {
  const tree = renderer.create(
    <Toast.Action>Action</Toast.Action>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});

test('Toast action passes props correctly', () => {
  const tree = renderer.create(
    <Toast.Action className="my-class" data-x="x" title="Action">
      Action
    </Toast.Action>
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
