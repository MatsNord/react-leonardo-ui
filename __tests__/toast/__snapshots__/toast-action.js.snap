// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Toast action passes props correctly 1`] = `
<div
  className="lui-toast__aside my-class"
  data-x="x"
  title="Action"
>
  Action
</div>
`;

exports[`Toast action renders correctly 1`] = `
<div
  className="lui-toast__aside undefined"
>
  Action
</div>
`;
